cscs-checks/libraries/gridtools/gridtools_base.py

```diff
@@ -0,0 +1,180 @@
+#                                         '-DGT_ENABLE_TARGET_NAIVE:BOOL=ON',
+#                                         '-DGT_ENABLE_TARGET_CUDA:BOOL=OFF',
+#                                         '-DGT_ENABLE_TARGET_MC=ON',
+#                                         '-DGT_GCL_ONLY:BOOL=OFF',
+#                                         '-DCMAKE_CXX_COMPILER=CC',
+#                                         '-DGT_USE_MPI:BOOL=OFF', 
+#                                         '-DGT_SINGLE_PRECISION:BOOL=OFF', 
+#                                         '-DGT_ENABLE_PERFORMANCE_METERS:BOOL=ON',
+#                                         '-DGT_TESTS_ICOSAHEDRAL_GRID:BOOL=OFF',
+#                                         '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON',
+#                                         '-DBOOST_ROOT=$BOOST_ROOT',
+#                                         '-DGT_ENABLE_PYUTILS=OFF',
+#                                         '-DGT_TESTS_REQUIRE_FORTRAN_COMPILER=ON',
+#                                         '-DGT_TESTS_REQUIRE_C_COMPILER=ON',
+#                                         '-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON']
+        self.build_system.config_opts = ['-DCMAKE_INSTALL_PREFIX=/scratch/snx3000/bignamic/reframe/cscs-checks/libraries/gridtools/install',
+                                         '-DBoost_NO_BOOST_CMAKE="true"',
+                                         '-DCMAKE_BUILD_TYPE:STRING=Release',
+                                         '-DBUILD_SHARED_LIBS:BOOL=ON',
+                                         '-DGT_ENABLE_TARGET_X86:BOOL=OFF',
+                                         '-DGT_ENABLE_TARGET_NAIVE:BOOL=OFF',
+                                         '-DGT_ENABLE_TARGET_CUDA:BOOL=ON',
+                                         '-DCUDA_ARCH:STRING=sm_60',
+                                         '-DGT_GCL_ONLY:BOOL=OFF',
+                                         '-DCMAKE_CXX_COMPILER=CC',
+                                         '-DCMAKE_CUDA_HOST_COMPILER:STRING=CC',
+                                         '-DGT_USE_MPI:BOOL=OFF', 
+                                         '-DGT_SINGLE_PRECISION:BOOL=OFF', 
+                                         '-DGT_ENABLE_PERFORMANCE_METERS:BOOL=ON',
+                                         '-DGT_TESTS_ICOSAHEDRAL_GRID:BOOL=OFF',
+                                         '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON',
+                                         '-DBOOST_ROOT=$BOOST_ROOT',
+                                         '-DGT_ENABLE_PYUTILS=OFF',
+                                         '-DGT_TESTS_REQUIRE_FORTRAN_COMPILER=ON',
+                                         '-DGT_TESTS_REQUIRE_C_COMPILER=ON',
+                                         '-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON']
+
+
+        self.sanity_patterns = sn.assert_found(r'PASSED', self.stdout)
+        self.perf_patterns = {
+            'perf': sn.extractsingle(r'(?P<timer>\w+) ms total',
```

Comment:
Do not name the performance variable `perf`. What is the thing your are timing.

Reply:
It is wall time, variable name updated.